src/dwg.spec

```diff
@@ -802,9 +802,7 @@ DWG_ENTITY (MINSERT)
 
 DWG_ENTITY_END
 
-/*
-// 9 was TRACE. r10-r11 only
```

Comment:
I'd like to keep the TRACE comment there

Reply:
I don't think that comment is true. But no problem with it.

Reply:
Ah entity id 9 for trace is in <AC1.50 - look https://github.com/michal-josef-spacek/CAD-Format-DWG-AC1_40/blob/main/dwg_ac1_40.ksy

FYI: format of DWG to <AC1.50 is one, from AC1.50 to AC1009 second and >AC1009 third. We could add decoding section for it. I know everything in AC1.40 instead of structure in 0x01d0-0x01d7 (i have 8 byte float, but i don't think that it is - converted to format  in AC1.50 and above. Seems as some structure).

Reply:
uff. I was looking to it. And trace is id 9 in all versions to <=AC1009. And i don't understand sense of this comment. This is comment for 3DLINE.